src/components/merge-map.js

```diff
@@ -75,8 +80,24 @@ const MergeMap = (props) => {
                 return status.replacingDate == null ? '#02538B' : '#F3D111';
             case IgmStatus.INVALID:
                 return status.replacingDate == null ? '#D8404D' : '#D86640';
-            case IgmStatus.MERGED:
-                return status.replacingDate == null ? '#37AE4B' : '#90EE90';
+            case IgmStatus.MERGED: {
+                switch (status.cgmStatus) {
+                    case CgmStatus.VALID:
+                        return status.replacingDate == null
+                            ? '#37AE4B'
+                            : '#90EE90';
+                    case CgmStatus.VALID_WARNING:
+                        return status.replacingDate == null
+                            ? '#FFA500'
+                            : '#FFC14E';
+                    case CgmStatus.INVALID:
+                        return status.replacingDate == null
+                            ? '#FF0000'
```

Comment:
maybe we can create some constants instead of directly injecting colors codes to know more about colors for each case:
exp:  RED_COLOR="#FF0000"

Reply:
Done